src/components/form-elements/draft-js-mention-selector/DraftJSMentionSelectorCore.js

```diff
@@ -285,6 +295,15 @@ class DraftJSMentionSelector extends React.Component<Props, State> {
                         : []}
                 </SelectorDropdown>
                 {showMentionStartState ? <MentionStartState message={startMentionMessage} /> : null}
+                {announceActiveMention && (
+                    <span className="accessibility-hidden" data-testid="accessibility-alert" role="alert">
+                        {this.shouldDisplayMentionLookup() ? (
+                            <UsersFoundMessage contactsCount={contacts.length} />
+                        ) : (
+                            <NoUsersFoundMessage />
```

Comment:
Are these thin wrappers around `FormattedMessage` needed? Could we simplify by using it directly? Maybe something like
```
const message = this.shouldDisplayMentionLookup() ? {...messages.usersFound, values: count: contacts.length} : messages.noUsersFound;
<FormattedMessage {...message} />
```

Reply:
done